ipc/cli/src/commands/subnet/join.rs

```diff
@@ -136,3 +147,45 @@ pub struct UnstakeSubnetArgs {
     )]
     pub collateral: f64,
 }
+
+pub struct PreFundSubnet;
+
+#[async_trait]
+impl CommandLineHandler for PreFundSubnet {
+    type Arguments = PreFundSubnetArgs;
+
+    async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("pre-fund subnet with args: {:?}", arguments);
+
+        let mut provider = get_ipc_provider(global)?;
+        let subnet = SubnetID::from_str(&arguments.subnet)?;
+        let from = match &arguments.from {
+            Some(address) => Some(require_fil_addr_from_str(address)?),
+            None => None,
+        };
+        provider
+            .prefund_subnet(
+                subnet.clone(),
+                from,
+                f64_to_token_amount(arguments.initial_balance)?,
+            )
+            .await?;
+        println!("address pre-funded successfully");
```

Comment:
```suggestion
        log::info!("address pre-funded successfully");
```

Reply:
Same as above, we are moving from logs in command returns.

Reply:
I don't think this is a real return value, not something you would want to pipe into another program. The only return value from this should be a 0 exit code. There is no variable data in it.